e2e-polybft/e2e/jsonrpc_test.go

```diff
@@ -25,447 +22,6 @@ var (
-		require.NoError(t, err)
-
-		// Test. The populated fields match with the block
-		block, err := client.GetBlockByHash(receipt.BlockHash, false)
-		require.NoError(t, err)
-
-		require.Equal(t, receipt.TransactionHash, txn.Receipt().TransactionHash)
-		require.Equal(t, receipt.BlockNumber, block.Number)
-		require.Equal(t, receipt.BlockHash, block.Hash)
-
-		// Test. The receipt of a deployed contract has the 'ContractAddress' field.
-		txn = cluster.Deploy(t, senderKey, contractsapi.TestSimple.Bytecode)
-		require.NoError(t, txn.Wait())
-		require.True(t, txn.Succeed())
-
-		require.NotEqual(t, txn.Receipt().ContractAddress, ethgo.ZeroAddress)
-	})
-
-	t.Run("eth_getTransactionByHash", func(t *testing.T) {
-		key1, err := crypto.GenerateECDSAKey()
-		require.NoError(t, err)
-
-		// Test. We should be able to query the transaction by its hash
-		txn := cluster.Transfer(t, senderKey, key1.Address(), one)
-		require.NoError(t, txn.Wait())
-		require.True(t, txn.Succeed())
-
-		ethTxn, err := client.GetTransactionByHash(txn.Receipt().TransactionHash)
-		require.NoError(t, err)
-
-		// Test. The dynamic 'from' field is populated
-		require.NotEqual(t, ethTxn.From, ethgo.ZeroAddress)
-	})
-
-	t.Run("eth_getHeaderByNumber", func(t *testing.T) {
-		key1, err := crypto.GenerateECDSAKey()
-		require.NoError(t, err)
-
-		txn := cluster.Transfer(t, senderKey, key1.Address(), one)
-		require.NoError(t, txn.Wait())
-		require.True(t, txn.Succeed())
-		txReceipt := txn.Receipt()
-
-		var header types.Header
-		err = jsonRPC.Call("eth_getHeaderByNumber", &header, ethgo.BlockNumber(txReceipt.BlockNumber))
-		require.NoError(t, err)
-
-		require.Equal(t, txReceipt.BlockNumber, header.Number)
-		require.Equal(t, txReceipt.BlockHash, ethgo.Hash(header.Hash))
-	})
-
-	t.Run("eth_getHeaderByHash", func(t *testing.T) {
-		key1, err := crypto.GenerateECDSAKey()
-		require.NoError(t, err)
-
-		txn := cluster.Transfer(t, senderKey, key1.Address(), one)
-		require.NoError(t, txn.Wait())
-		require.True(t, txn.Succeed())
-		txReceipt := txn.Receipt()
-
-		var header types.Header
-		err = jsonRPC.Call("eth_getHeaderByHash", &header, txReceipt.BlockHash)
-		require.NoError(t, err)
-
-		require.Equal(t, txReceipt.BlockNumber, header.Number)
-		require.Equal(t, txReceipt.BlockHash, ethgo.Hash(header.Hash))
-	})
-
-	t.Run("debug_traceTransaction", func(t *testing.T) {
-		key1, err := crypto.GenerateECDSAKey()
-		require.NoError(t, err)
-
-		txn := cluster.Transfer(t, senderKey, key1.Address(), one)
-		require.NoError(t, txn.Wait())
-		require.True(t, txn.Succeed())
-
-		txReceipt := txn.Receipt()
-
-		// Use a wrapper function from "jsonrpc" package when the config is introduced.
-		trace, err := debug.TraceTransaction(txReceipt.TransactionHash, jsonrpc.TraceTransactionOptions{})
-		require.NoError(t, err)
-		require.Equal(t, txReceipt.GasUsed, trace.Gas)
-	})
-}
-
-func TestE2E_JsonRPC_NewEthClient(t *testing.T) {
```

Comment:
Rename it now to `TestE2E_JSONRPC`?

Reply:
Fix: https://github.com/Ethernal-Tech/blade/pull/165/commits/6a43df12d00f176d42daa16a06305926806ccfcd